### PR TITLE
UPGRADE MONGOID

### PR DESCRIPTION
**Pull request by chigginsboost**
This change is needed for this AQ story: https://www.pivotaltracker.com/story/show/187737365

We need to upgrade mongoid in consyncful before we can update it in AQ. This story bumps the mongoid version to 9

### CODE REVIEW

**Comment by chigginsboost** [comment]
for reviewers: anyone know why the checks started failing on the version bump? I feel like I did something wrong here. Thanks!

**Comment by motizuki** [comment]
Nice work, is this a breaking change? if so we should probably be a major version bump in this gem.

**Review by motizuki — APPROVED** [review]

File: lib/consyncful/base.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'consyncful/configuration'
```
Comment (hapiben) [inline lib/consyncful/base.rb:3]: **question:** Will it work without this `require`?

Comment (chigginsboost) [reply lib/consyncful/base.rb:3]: not sure, I can give it a go 👍

**Review by hapiben — APPROVED** [review]

File: lib/consyncful/sync.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
 require 'consyncful/item_mapper'
 require 'consyncful/persisted_item'
 require 'consyncful/stats'
+require 'consyncful/configuration'
```
Comment (hapiben) [inline lib/consyncful/sync.rb:7]: Same here. If it doesn't work. That's all good.

Comment (chigginsboost) [reply lib/consyncful/sync.rb:7]: I think it does! tests are still passing :) I've removed from both base and sync

**Pull request merged**